types/pretty-time/index.d.ts

```diff
@@ -0,0 +1,40 @@
+// Type definitions for pretty-time 1.1
+// Project: https://github.com/jonschlinkert/pretty-time
+// Definitions by:  Adam Zerella <https://github.com/adamzerella>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.3
+
+export type Increment =
+    | 'ns'
+    | 'nano'
+    | 'nanosecond'
+    | 'nanoseconds'
+    | 'μs'
+    | 'micro'
+    | 'microsecond'
+    | 'microseconds'
+    | 'ms'
+    | 'milli'
+    | 'millisecond'
+    | 'milliseconds'
+    | 's'
+    | 'sec'
+    | 'second'
+    | 'seconds'
+    | 'm'
+    | 'min'
+    | 'minute'
+    | 'minutes'
+    | 'h'
+    | 'hr'
+    | 'hour'
+    | 'hours'
+    | 'd'
+    | 'day'
+    | 'days'
+    | 'w'
+    | 'wk'
+    | 'week'
+    | 'weeks';
+
+export default function(time: number[], smallest?: Increment, digits?: number): string;
```

Comment:
It should be `export =`: https://github.com/jonschlinkert/pretty-time/blob/master/index.js#L12

Reply:
@plantain-00 I'm not sure how to export a default function with `export = `. Should I be declaring a `module` ?

Reply:
See:
http://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-function-d-ts.html
https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default

Reply:
Ahh gotcha! updated 😃